tests/host_tools/cargo_build.py

```diff
@@ -45,7 +45,7 @@ def cargo_test(path, extra_args=''):
     path = os.path.join(path, CARGO_UNITTEST_REL_PATH)
     cmd = 'CARGO_TARGET_DIR={} RUST_TEST_THREADS=1 RUST_BACKTRACE=1 ' \
           'RUSTFLAGS="{}" cargo test {} --all --no-fail-fast'.format(
-            path, get_rustflags(), extra_args)
+              path, get_rustflags(), extra_args)
```

Comment:
Was this needed?

Reply:
in my opinion, not. I see that the rust formatter in VSCode added this automatically